extensions/provisioning/creates.go

```diff
@@ -536,23 +536,29 @@ func CreateProvisioningRKE1CustomCluster(client *rancher.Client, externalNodePro
 // CreateProvisioningAirgapCustomCluster provisions a non-rke1 cluster using corral to gather its nodes, then runs verify checks
 func CreateProvisioningAirgapCustomCluster(client *rancher.Client, clustersConfig *clusters.ClusterConfig, corralPackages *corral.Packages) (*v1.SteveAPIObject, error) {
 	setLogrusFormatter()
-	rolesPerNode := map[int32]string{}
```

Comment:
Question why change this from a map? Isn't it much more efficient doing that rather than keeping track of two data structures?

Reply:
this is matching what we have for all the other createCustomCluster functions. I just changed it to what I knew was working, since currently airgap wasn't working for multi-pool.